### PR TITLE
Portfolio: accessibility

### PR DESCRIPTION
**Pull request by BereketRetta**
This is the report for the Accessibility Checks.
- **Page titles** - issues found and fixed it as you can see from my commit.
- **mage text alternatives** - some images didn't have `alt`  as well fixed by my commit "FIXED TITLE AND IMG ALT FOR ACCESSIBILITY";
- **Text headings** -  no issues found;
- **Color contrast** - my contrast ratio is 8.59:1 which is well above the required.
- **Resize** -  no issues found
- **Interaction**-  no issues found
- **Moving content** -  not found some animations but those take 1s to complete.
- **Multimedia**-  not found
- **The basic structure of the page** -  it is very well structured

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Portfolio</title>
+    <title>Personal Portfolio - Bereket Retta's Personal Portfolio Website</title>
 
     <link
```
Comment (andersonlebon) [inline index.html:9]: - [ ] The requirement of this project says that the students should write a report about the accessibility of their portfolio, based on the Easy Checks – A First Review of Web Accessibility article, listing the accessibility points they checked and writing a conclusion summarizing the results of the evaluation.  Just make a summary of all the accessibilities found in your project and put it in your pull request. Kindly you can follow the template :point_down: :point_down: 

````
- **Page titles** - no issues found;
- **mage text alternatives** - some images didn't have `alt` text [link](link);
- **Text headings** -  no issues found;
- **Color contrast** - your report
- **Resize** -  your report
- **Interaction**-  your report
- **Moving content** -  your report
- **Multimedia**-  not found
- **The basic structure of the page** -  your report
- etc..
````

**Review by andersonlebon — CHANGES_REQUESTED** [review]
### Hi @BereketRetta 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (BereketRetta) [reply index.html:9]: Thank you @andersonlebon for explaining but I am confused about where should I put the explanations? should I put them in the description ?.

Comment (andersonlebon) [reply index.html:9]: Hello @BereketRetta  you can just edit your pull request description and put the summary inside. thanks

**Comment by ShoiraTa** [comment]
### **STATUS: APPROVED  ✔️** 
------
Hi @BereketRetta ,  

Your project is complete! 🎉💯🌟 




Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**